tests/ecosystem/upgrade/test_logging_upgrade.py

```diff
@@ -161,6 +171,9 @@ def upgrade_info(channel):
 
 
 @post_ocp_upgrade
+@pytest.mark.usefixtures(
+    setup_fixture.__name__
```

Comment:
Why you wrap the fixtrue install_logging by setup_fixture and not directly using install_logging here? If it's about logging message, this logging message should be written from install_logging IMO.

Reply:
Done